### PR TITLE
Bumps azure/login from 1 to 2.2.0

### PR DESCRIPTION
**Pull request by shanice-skylight**
# DEVOPS PULL REQUEST

## Related Issue

- While updating credentials for Azure and terraform, noticed that azure/login was behind on an update an not consistent in all modules. 
- Bumps [azure/login](https://github.com/azure/login) from 1 to 2.2.0.

## Changes Proposed

- Modify all azure/login version to 2.2.0

## Additional Information
 
- Deployed to `dev2` to tet it.

## Testing

- How should reviewers verify this PR?

<!---
### Infrastructure
- [ ] Consult the results of the `terraform-plan` job inside the "Terraform Checks" workflow run for this PR. Confirm that there are no unexpected changes!

### CODE REVIEW

File: .github/workflows/slotSwapStgAPI.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       ARM_TENANT_ID: ${{ secrets.TERRAFORM_ARM_TENANT_ID }}
     steps:
       - uses: actions/checkout@v4
-      - uses: azure/login@v2
+      - uses: azure/login@v2.2.0
```
Comment (emyl3) [inline .github/workflows/slotSwapStgAPI.yml:26]: Question for my own understanding... Is there a reason we want to specify the minor version in this change? It looks like `v2` tag without a minor version specified points to [v2.2.0](https://github.com/Azure/login/releases/tag/v2) 🤔

**Review by emyl3 — COMMENTED** [review]
Left one question for you, Shanice! Thank you for catching this!

**Review by DavidMcClatchey — APPROVED** [review]
LGTM!

**Review by mpbrown — APPROVED** [review]
LGTM!

File: .github/workflows/slotSwapStgAPI.yml (hunk 1 above)
Comment (shanice-skylight) [reply .github/workflows/slotSwapStgAPI.yml:26]: I didn't realize that `v2` tag already pointed to `2.2.0`.  I'll remove the minor version so they are all consistent.

**Comment by shanice-skylight** [comment]
Waiting for cert renewal to be complete, then will move forward with merge

**Review by emyl3 — APPROVED** [review]
Thank you!! LGTM!

**Review by rin-skylight — APPROVED** [review]

**Comment by shanice-skylight** [comment]
Re-ran backend [test](https://github.com/CDCgov/prime-simplereport/actions/runs/12555168174/job/35005128209) after it was removed from the merge queue and the test passed, so re-adding it to the merge queue.

**Pull request merged**